README.rst

```diff
@@ -1144,8 +1144,37 @@ When operating on stdin, use the ``--input-type`` and ``--output-type`` flags as
 
     $ cat myfile.json | sops --input-type json --output-type json -d /dev/stdin
 
+JSON indentation
+~~~~~~~~~~~~~~~~
+
+``sops`` indent ``SOPS`` files by default using one ``tab``. However, you can change
```

Comment:
Uh, I believe this is supposed to be "sops indents *JSON* files", no?

Reply:
Nice catch, thank you !